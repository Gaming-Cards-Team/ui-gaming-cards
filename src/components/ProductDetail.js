import React from "react";
import Row from "react-bootstrap/Row";
import Col from "react-bootstrap/Col";

const imageStyle = {
  textAlign: "right",
};
const detailStyle = {
  color: "#2d963f",
  fontWeight: "bold",
};

const ProductDetail = () => (
  <div>
    <Row>
      <Col xs={5} style={imageStyle}>
        
      </Col>
      <Col xs={7} style={detailStyle}>
        
      </Col>
    </Row>
  </div>
);

export default ProductDetail;
