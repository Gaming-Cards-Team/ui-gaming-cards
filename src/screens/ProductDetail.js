import React from "react";
import Header from "../components/Header";
import ProductDetailBody from "../components/ProductDetailBody";
import Footer from "../components/Footer";

const ProductDetail = () => {
  return (
    <>
      <Header />
      <ProductDetailBody />
      <Footer />
    </>
  );
};

export default ProductDetail;
