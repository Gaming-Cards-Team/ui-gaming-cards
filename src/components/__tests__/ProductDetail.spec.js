import React from 'react';
import ProductDetail from '../ProductDetail';

import renderer from 'react-test-renderer';

describe('Test Product Detail component', () => {
    it('should validate the component', () => {
        const productDetail = renderer.create(<ProductDetail />).toJSON();

        expect(productDetail).toMatchSnapshot();
    });
});