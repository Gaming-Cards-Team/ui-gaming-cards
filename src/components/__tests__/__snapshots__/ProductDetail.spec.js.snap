// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Product Detail component should validate the component 1`] = `
<div>
  <div
    className="row"
  >
    <div
      className="col-5"
      style={
        Object {
          "textAlign": "right",
        }
      }
    />
    <div
      className="col-7"
      style={
        Object {
          "color": "#2d963f",
          "fontWeight": "bold",
        }
      }
    />
  </div>
</div>
`;
